### ISSUE TITLE
Mouse scroll wheel doesn't move volume slider

### ISSUE THREAD

**Issue by Uroshi** [maciex/play-to-kodi#19]
Hello

My configuration:
Windows 7 Home Premium 64bit
Firefox 57.0.2 64 bit

As I have mentioned in the subject, mouse scroll wheel doesn't move volume slider.
This isn't happening with Chrome on the same system

Best regards

**Comment by maciex** [comment]
Yes, the problem exists in Firefox.
It's probably caused by the '[mousewheel](https://developer.mozilla.org/en-US/docs/Web/Events/mousewheel)' event being deprecated:

> Non-standard
> This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

I'll try to rewrite the code to '[wheel](https://developer.mozilla.org/en-US/docs/Web/Events/wheel)' event.

**Comment by maciex** [comment]
@Uroshi I just fixed the code and uploaded new beta version 1.9.1beta2 to Addons Mozilla.

Please try the beta version and let me know, if it works for you.
https://addons.mozilla.org/en-US/firefox/addon/play-to-kodi4/versions/beta

**Comment by Uroshi** [comment]
I have tried beta 2 with current Firefox (57.0.4) but I can't connect to Kodi... so I can't test volume slider...
And yes, I can connect to Kodi using Play to Kodi 1.9.0...

**Comment by maciex** [comment]
That's unexpected.
When on 1.9.1.beta2, do you have all the settings in the hosts fields?
Could you enable debug logs in the settings and check the logs?

**Comment by Uroshi** [comment]
Yes, I input URL and Port values but when I click on save there isn't green message "Options Saved" which should be displayed bellow Save button... And when I click on Back button there is still orange message: Connecting...

No, I can't Enable Debug Logs because Save button doesn't work in this segment also...

I was in a hurry and didn't provide detailed information... I hope my this post will help you fix the issue.

**Comment by Uroshi** [comment]
I was in a hurry and didn't provide detailed information... I hope my last post will help you fix the issue.

**Comment by maciex** [comment]
I can't reproduce the problem.

If that's possible please follow these steps to debug the problem:

1. Exit and run Firefox again
2. Enable Add-on debugging for Play to Kodi Add-on, using instruction from Mozilla:
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Debugging#The_Add-on_Debugger
3. In the debug "Console" run the following commands and provide output here.
**I will can contain hosts/usernames/passwords, so please replace them if needed!**

- This will show the settings in the old format (used in v1.9.0 and before):
`JSON.stringify(localStorage);`
- This will show the setting in the new format (used all new versions above v1.9.0):
`browser.storage.sync.get().then(settings => console.log(JSON.stringify(settings)));`
- This will set the debugging:
`browser.storage.sync.set({enableDebugLogs: true});`
4. Go to the Play to Kodi Settings and try to edit and save the Settings.
5. Check output in Console and provide it here.

**Comment by Uroshi** [comment]
I have tried beta 2 on another PC. Also Windows  7 64 bit and Firefox 57.0.4 64 bit and mouse wheel volume slider works!

So this problem with save button is some my local problem...

Thanks

**Comment by maciex** [comment]
@Uroshi That's good it's working after the fix.

If you would like to debug the problem with 1.9.1beta2 on the first system, please follow the instructions provided to gather the data, then open separate Issue.
Thanks!

Fixed in dfbc7fb46fd6ba7f017d780de763d0063e2d11e3
